src/styles/blocks/place.scss

```diff
@@ -0,0 +1,51 @@
+.place {
+  &__photo-container {
+    position: relative;
+
+    box-sizing: border-box;
+    width: 100%;
+    max-height: 256px;
```

Comment:
According to the design, these pictures should have less height on mobile screens. Now they always have 256px. Fix this